checks/permissions.go

```diff
@@ -236,13 +252,19 @@ func calculateScore(result permissionCbData) int {
 	// Note: there are legitimate reasons to use some of the permissions like checks, deployments, etc.
 	// in CI/CD systems https://docs.travis-ci.com/user/github-oauth-scopes/.
 
-	if permissionIsPresent(result, "all") {
-		return checker.MinResultScore
-	}
-
 	// Start with a perfect score.
 	score := float32(checker.MaxResultScore)
 
+	// If no top level permissions are defined...
+	if permissionIsPresentInTopLevel(result, "all") {
```

Comment:
Similar comment here. Comment says `no top level permissions are defined` but the fn checks for `permissionIsPresentInTopLevel`.

Reply:
let me update the comment. Thanks!